telemetry/main.schema.json

```diff
@@ -33,6 +33,10 @@
           "type": "string",
           "pattern": "^\\d{2}\\."
         },
+        "displayVersion": {
+          "type": "string",
+          "pattern": "^\\d{2}\\."
```

Comment:
yes, we should move to 3 digits to avoid potential issues, it is in a few years (about 6 if we keep the same rhythm).